conf/application.conf

```diff
@@ -117,6 +122,7 @@ urls {
   hmrcCallCharges           = "https://www.gov.uk/call-charges"
   incomeTaxGeneralEnquiries = "https://www.gov.uk/government/organisations/hm-revenue-customs/contact/income-tax-enquiries-for-individuals-pensioners-and-employees"
   employeeExpensesUrl       = "http://localhost:9334"
+  basGatewaySignOut         = "http://localhost:9553/bas-gateway/sign-out-without-state"
```

Comment:
Nit: Could you change it to sth like `basGatewayFrontendSignOutUrl` ? To distinguish between FE and BE.